Sources/SwiftDocC/DocumentationService/Convert/ConvertService.swift

```diff
@@ -286,10 +292,13 @@ public struct ConvertService: DocumentationService {
             }
 
         var baseStore = baseReferenceStore ?? RenderReferenceStore()
-        baseStore.topics.merge(topicContent, uniquingKeysWith: { old, _ in
-            // Prioritize content that was in the base store, it might be more accurate.
-            return old
-        })
+        baseStore.topics.merge(
+            topicContent,
+            uniquingKeysWith: { old, _ in
```

Comment:
here's another example of the same odd line wrapping

Reply:
I think it might be this setting: `"lineBreakBeforeEachArgument" : true,`

Reply:
If this used trailing closure syntax I bet it wouldn't reflow it.